contrib/bdii/htcondor-ce-provider

```diff
@@ -208,6 +216,11 @@ def main():
         except (KeyError, RuntimeError):
             state = 'critical'
             state_info = 'Authorization ping failed'
+        except Exception, e:
```

Comment:
Let's catch either `RuntimeError` (and verify the TimeoutError message, raising if the message doesn't match) or `TimeoutException` depending on what you do above.

Reply:
Going to catch the TimeoutError Exception.